packages/create-cloudflare/e2e-tests/workers.test.ts

```diff
@@ -178,18 +178,6 @@ function getWorkerTests(opts: { experimental: boolean }): WorkerTestConfig[] {
 				verifyDeploy: null,
 				verifyPreview: null,
 			},
-			{
-				template: "openapi",
-				variants: [],
-				verifyDeploy: {
-					route: "/",
-					expectedText: "SwaggerUI",
-				},
-				verifyPreview: {
-					route: "/",
-					expectedText: "SwaggerUI",
-				},
-			},
```

Comment:
Can you either comment out and leave a TODO; or create an issue to add this back in.

Reply:
I'm going to create a revert PR immediately after this gets merged